When built against PUC Lua, minetest binaries get linked against
a customised version of LuaBitOp which then gets installed
as /usr/$(get_libdir)/libbitop.so. This violates the policies of both
Gentoo (unversioned shared library) and Lua itself (compiled extensions
should be installed into implementation-specific directories rather than
top-level libdir, and I'm pretty sure linking against such extensions
is a no-no). Switching to system dev-lua/LuaBitOp will require work
so just make the customised version static for now.

--- a/lib/bitop/CMakeLists.txt
+++ b/lib/bitop/CMakeLists.txt
@@ -1,4 +1,4 @@
-add_library(bitop bit.c)
+add_library(bitop STATIC bit.c)
 target_link_libraries(bitop)
 
 include_directories(${LUA_INCLUDE_DIR})
